x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/store/input/FrozenIndexInputTests.java

```diff
@@ -83,7 +85,7 @@ public void testRandomReads() throws IOException {
             cacheSize = ByteSizeValue.ofBytes(randomLongBetween(1L, 10L) * regionSize.getBytes() + randomIntBetween(0, 100));
         }
 
-        final Settings settings = Settings.builder()
+        final Settings settings1 = Settings.builder()
```

Comment:
What am I missing here, why do we need that redundant variable?

Reply:
Thanks, that makes no sense, is a leftover from refactoring this in an earlier iteration on it.